src/test/java/uk/gov/hmcts/reform/sscs/helper/HearingsDetailsMappingTest.java

```diff
@@ -262,10 +262,8 @@ void getHearingLocations(String baseLocation, String region,
         ",,Normal"
     }, nullValues = {"null"})
     void getHearingPriority(String isAdjournCase, String isUrgentCase, String expected) {
```

Comment:
I would change `String expected` to `HearingPriority expected` and change the csv to the Enum values (uppercase), then use .getKey() on the assert

Reply:
I tried doing this but can't pass enum into the CsvSourse